Code/Framework/AzQtComponents/AzQtComponents/Components/StyleManager.cpp

```diff
@@ -181,6 +238,14 @@ namespace AzQtComponents
         m_autoCustomWindowDecorations = new AutoCustomWindowDecorations(this);
         m_autoCustomWindowDecorations->setMode(AutoCustomWindowDecorations::Mode_AnyWindow);
 
+        AZStd::string themeName = "O3DE_Original";
+
+        AZStd::string themePropertiesPath = AZStd::string::format(
+            "D:\\GitRepos\\O3DE\\o3de\\Code\\Framework\\AzQtComponents\\AzQtComponents\\Themes\\%s\\themeProperties.json",
+             themeName.c_str());
```

Comment:
Find this without hard-coding the path. Might need to go under `Assets/Editor` so it can be found by the installer as well?